src/usr/share/netcheck/networkmanagerhelper.py

```diff
@@ -413,6 +420,28 @@ def connection_is_activated(self, connection_id):
 
         return connection_is_activated
 
+    def get_connection_for_interface(self, interface_name):
+        """Find and return the connection id currently applied to the given interface.
+
+        interface_name: The system interface name, for example eth1 or ens0s1.
```

Comment:
I'd probably put eth1 and ens0s1 in single quotes, but this is simply a matter of preference. Ignore me.